Void requires some tweaks to the default upstream PAM configuration.

--- a/unix/vncserver/tigervnc.pam	2023-02-28 10:11:20.000000000 -0500
+++ b/unix/vncserver/tigervnc.pam	2023-07-30 17:54:21.281000000 -0400
@@ -1,11 +1,11 @@
 #%PAM-1.0
 # pam_selinux.so close should be the first session rule
--session   required     pam_selinux.so close
+-session   optional     pam_selinux.so close
 session    required     pam_loginuid.so
--session   required     pam_selinux.so open
+-session   optional     pam_selinux.so open
 session    required     pam_namespace.so
 session    optional     pam_keyinit.so force revoke
 session    required     pam_limits.so
--session   optional     pam_systemd.so
+-session   optional     pam_elogind.so
 session    required     pam_unix.so
 -session   optional     pam_reauthorize.so prepare
